skyportal/handlers/api/internal/standards.py

```diff
@@ -0,0 +1,108 @@
+from ...base import BaseHandler
+from ....utils.offset import get_formatted_standards_list
+
+
+class StandardsHandler(BaseHandler):
+    @auth_or_token
+    def get(self):
+        """
+        ---
+        description: Get standard stars with specified formatting
+        parameters:
+        - in: query
+          name: facility
+          nullable: true
+          required: false
+          schema:
+            type: string
+            enum: [Keck, Shane, P200]
+          description: Which facility to generate the starlist for
+        - in: query
+          name: standard_type
+          required: false
+          schema:
+            type: string
+          description: |
+            Origin of the standard stars, defined in config.yaml
+        - in: query
+          name: dec_filter_range
+          nullable: True
+          required: false
+          schema:
+            type: list
+          description: |
+            lowested and highest dec to return
+        - in: query
+          name: ra_filter_range
+          required: false
+          nullable: True
+          schema:
+            type: list
+          description: |
+            lowested and highest dec to return (or wrapped range)
+        - in: query
+          name: show_first_line
+          required: false
+          schema:
+            type: boolean
+          description: |
+            return the first line of if demanded by the format
+        responses:
+          200:
+            content:
+              application/json:
+                schema:
+                  - $ref: '#/components/schemas/Success'
+                  - type: object
+                    properties:
+                      data:
+                        type: object
+                        properties:
+                          success:
+                            type: boolean
+                            description: did we get back a starlist as we expect?
+                          starlist_info:
+                            type: array
+                            description: |
+                              list of source and offset star information
+                            items:
+                              type: object
+                              properties:
+                                str:
+                                  type: string
+                                  description: single-line starlist format per object
+          400:
+            content:
+              application/json:
+                schema: Error
+        """
+        starlist_type = self.get_query_argument('facility', 'Keck')
+        standard_type = self.get_query_argument('standard_type', 'ESO')
+        dec_filter_range_str = self.get_query_argument('dec_filter_range', "[-90, 90]")
+        ra_filter_range_str = self.get_query_argument('ra_filter_range', "[0, 360]")
+        show_first_line = self.get_query_argument('show_first_line', False)
```

Comment:
Validate these

Reply:
I.e., the passed values themselves. Should reject stuff like:

`[a, 214.]`
`[None, None]`
`[-5000, 5000]`
 
etc.

Reply:
Done.

Reply:
what happens if someone posts an RA outside 0 - 360 or a dec outside -90 to 90 ?

Reply:
Now doing range checking.